enhancements/installer/cluster-api-based-installations.md

```diff
@@ -0,0 +1,347 @@
+---
+title: cluster-api-based-installations
+authors:
+  - "@patrickdillon"
+  - "@vincepri"
+  - "@JoelSpeed"
+reviewers:
```

Comment:
if you would like to be added here, please LMK

Reply:
I assume @avishayt or @mhrivnak should be reviewer to verify it does not affect assisted installer logic
cc: @oourfali @filanov